typespec-tests/tsp/naming.tsp

```diff
@@ -75,9 +75,10 @@ enum TypesModel {
 @friendlyName("DataStatus")
 union StatusModel {
   string,
-  "Running",
-  "Completed",
-  "Failed",
+  @clientName("LroRunning")
+  Running: "Running",
```

Comment:
Just a test for `@clientName`.

TCGC likely have bug on this case. Issue raised by Chenjie.